AWS/delete_iam_user.ipynb

```diff
@@ -0,0 +1,1022 @@
+{
```

Comment:
<p>This text doesnt sound right, this runbook is for delete user.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unskript/Awesome-CloudOps-Automation/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
it is not clear the text you are highlighting here, Which Action is the issue?

Reply:
updated the comment section.